src/bundle/Resources/public/js/scripts/sidebar/btn/content.edit.js

```diff
@@ -0,0 +1,18 @@
+(function () {
+    const editButton = document.querySelector('.ez-btn--edit');
+    const languageRadioOptions = document.querySelectorAll('.ez-extra-actions--edit [type="radio"]');
+
+    if (languageRadioOptions.length !== 1) {
+        return;
+    }
+
+    editButton.addEventListener('click', () => {
+        const triggerOnClickEventForLanguage = (radioOption) => {
+            radioOption.checked = true;
+            radioOption.dispatchEvent(new CustomEvent('change'))
+        };
+
+        triggerOnClickEventForLanguage(languageRadioOptions[0]);
```

Comment:
With my previous comments you will have only one radioOption są this method is not needed and you can do just:
```js
radioOption.checked = true;
radioOption.dispatchEvent(new CustomEvent('change'));
```